1000mm/step6/index.md

```diff
@@ -290,37 +290,38 @@ diagram: "wiringDiagram750placeholder.jpg"
 <!-- end z-probe -->
 
 <img src="./photo/jpfs_DSC2873.jpg">
-<p>Place the spindle power cord, z-axis stepper motor wire, x-axis stepper motor wire, z-axis limit switch wire, x-axis limit switch wire, and z-probe wire into the drag chain. Close up the drag chain hinges.</p>
+<p>Place the spindle power cord, z-axis stepper motor wire, x-axis stepper motor wire, z-axis limit switch wire, x-axis limit switch wire, and z-probe wire into the drag chain. Take note of the wiring orientation. Each wire will have a connector and an exposed wire end. The connectors should be placed together at the top of the drag chain and the exposed wire at the bottom of the drag chain. Close up the drag chain hinges.</p>
 <img src="./photo/jpfs_DSC2883.jpg">
 <p>Wire up z-axis and x-axis stepper motors and limit switches.  Make sure you attach the red wire to the middle pin and the black wire to the hinged side of the switch.  The stepper motors can only be wired in one way, so if it doesn't fit, flip it over and try again.</p>
 <img src="./photo/P4220500jpg05.jpg">
-<p>Attach the drag chain end to the drag chain bracket on the x-carriage.</p>
+<p>Attach the drag chain end to the drag chain bracket on the x-carriage using two M5 x 12mm flat head cap screws and two M5 nylon insert lock nuts.</p>
 <img src="./photo/P4220503jpg08.jpg">
 <p>Attach the length of drag chain to the drag chain bracket installed on the aluminum extrusion.</p>
 <img src="./photo/jpfs_DSC2902.jpg">
-<p>Try using a screw driver to pry open the length of drag chain.  The drag chain end might feel like it's going to break, but it's very hard plastic and should hold up just fine.  This might be the most difficult part of the entire build.</p>
+<p>Try using a screw driver to pry open the length of drag chain. The drag chain end might feel like it's going to break, but it's very hard plastic and should hold up just fine.  This might be the most difficult part of the entire build.</p>
 <img src="./photo/jpfs_DSC2916.jpg">
-<p>Place the zip tie mounts in the aluminum extrusion slot and rotate them to secure in place.</p>
+<p>Place the zip tie mounts in the aluminum extrusion slot and rotate them to secure in place. You can also use the 10mm wrench to easily rotate them.</p>
```

Comment:
```suggestion
<p>Place the zip tie mounts in the aluminum extrusion slot and rotate them to secure in place. You can also insert an Allen wrench through the openings of the zip tie mount to use as leverage and more easily rotate.</p>
```

Reply:
@giovannadf Think we might need a bit more description as to how they would use the wrench.